### DISCUSSION TITLE
req.body disappeared ?

### DISCUSSION THREAD

**Discussion by MedLabs** [HapticX/happyx#368] [General]
My app worked with some post routes using req.body
but since I updated to 4.6 I get errors
how to use the request body ?

**Comment by Ethosa** [comment]
https://github.com/HapticX/happyx/issues/358

**Comment by MedLabs** [comment]
could you please provide an example how to use it ?

like :
```
post: "/users":
  return %*addUser(body())
```

**Reply by Ethosa** [reply]
> could you please provide an example how to use it ?
> 
> like :
> 
> ```
> post: "/users":
>   return %*addUser(body())
> ```

```nim
post "/users":
  return req.body.get()
```

**Reply by MedLabs** [reply]
There is no mention of this in the documentation

**Reply by Ethosa** [reply]
> There is no mention of this in the documentation

https://hapticx.github.io/happyx/#/guide/ssr_basics
